### ISSUE TITLE
test_indexvalues.BuffersizeMultipleChunksize fails on win-amd64-py3.5

### ISSUE THREAD

**Issue by cgohlke** [PyTables/PyTables#506]
```
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
PyTables version:    3.2.2
HDF5 version:        1.8.15-patch1
NumPy version:       1.9.3
Numexpr version:     2.4.4 (using VML/MKL 11.3.0)
Zlib version:        1.2.8 (in Python interpreter)
LZO version:         2.09 (Feb 04 2015)
BZIP2 version:       1.0.6 (6-Sept-2010)
Blosc version:       1.4.4 (2015-05-05)
Blosc compressors:   blosclz (1.0.2.1), lz4 (1.7.1), lz4hc (1.7.1), snappy (1.1.1), zlib (1.2.8)
Cython version:      0.23.2
Python version:      3.5.0 (v3.5.0:374f501f4567, Sep 13 2015, 02:27:37) [MSC v.1900 64 bit (AMD64)]
Platform:            Windows-10.0.10240
Byte-ordering:       little
Detected cores:      8
Default encoding:    utf-8
Default FS encoding: mbcs
Default locale:      (en_US, cp1252)
-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
<snip>
======================================================================
FAIL: None (tables.tests.test_indexvalues.BuffersizeMultipleChunksize)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "X:\Python35\lib\site-packages\tables\tests\test_indexvalues.py", line 3339, in test01
    numpy.testing.assert_almost_equal(result, res)
  File "X:\Python35\lib\site-packages\numpy\testing\utils.py", line 474, in assert_almost_equal
    return assert_array_almost_equal(actual, desired, decimal, err_msg)
  File "X:\Python35\lib\site-packages\numpy\testing\utils.py", line 842, in assert_array_almost_equal
    precision=decimal)
  File "X:\Python35\lib\site-packages\numpy\testing\utils.py", line 620, in assert_array_compare
    raise AssertionError(msg)
AssertionError:
Arrays are not almost equal to 7 decimals

(shapes (29,), (2,) mismatch)
 x: array([-20000, -19999, -19768, -19387, -19253, -19177, -19152, -19075,
       -19065, -19042, -18713, -18394, -17948, -17369, -17076, -16961,
       -16623, -16532, -16434, -16420, -16403, -16231, -16165, -16103,
       -15994, -15874, -15746, -15543, -15138], dtype=int64)
 y: array([-20000, -19999], dtype=int64)

----------------------------------------------------------------------
```

**Comment by andreabedini** [comment]
I tried multiple times but I can't reproduce it on OS X (all same versions)

**Comment by andreabedini** [comment]
debugging `Row.__next__indexed` I found this:
```
self.indexvalues.size 12
nchunksread 629
self.indexvalues.size 11
nchunksread 633
self.indexvalues.size 16358 <-- bogus
nchunksread 637
self.indexvalues.size 17
nchunksread 640
self.indexvalues.size 7
```

so that's where things for wrong.

**Comment by tomkooij** [comment]
Was fixed in #539
